### ISSUE TITLE
No way to get executed SQL or sproc output

### ISSUE THREAD

**Issue by craigahobbs** [zillow/ctds#8]
1. Consider the following script:

```
import ctds

connection = ctds.connect('localhost', user='sa', password='password')
with connection:
    with connection.cursor() as cursor:
        cursor.execute('''\
print('hello')
print('goodbye')
''')
```

Question: How do I get the "hello\ngoodbye\n" output?

**Comment by joshuahlang** [comment]
Currently there is no way to access messages returned to the client. I can update ctds to save these messages and expose them as a property on the connection object.

**Comment by joshuahlang** [comment]
Fixed in 1.4.0
